src/toolkits/object_detection/object_detector.hpp

```diff
@@ -40,8 +40,12 @@ class EXPORT object_detector: public ml_model_base {
   void train(gl_sframe data, std::string annotations_column_name,
              std::string image_column_name, variant_type validation_data,
              std::map<std::string, flexible_type> opts);
-  variant_map_type evaluate(gl_sframe data, std::string metric);
-  gl_sarray predict(gl_sframe data);
+  variant_type evaluate(gl_sframe data, std::string metric,
+                        std::string output_type = "dict",
+                        float confidence_threshold = 0.001,
+                        float iou_threshold = 0.45);
+  gl_sarray predict(variant_type data, float confidence_threshold,
+   float iou_threshold);
```

Comment:
I think the new arguments should probably be in an `options` dictionary, like what we do for `train`. That also makes it easier to define defaults that can change over time, if needed.

Reply:
:)